Added endpoint url to API initialization code

ryanshripat opened the pull request:
Calling new FaceServiceRestClient() without the endpoint specified results in "Invalid Subscription" errors depending on what endpoint was assigned by default. Explicitly specifying the endpoint is safer and is more likely to work for users of this tutorial.

azurecla commented:
Hi __@yamatoya__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by Microsoft and real humans are currently evaluating your PR.</p>

TTYL, AZPRBOT;

ryanshripat committed:
Added endpoint url to API initialization code

Calling new FaceServiceRestClient() without the endpoint specified results in "Invalid Subscription" errors depending on what endpoint was assigned by default. Explicitly specifying the endpoint is safer and is more likely to work for users of this tutorial.

GitHubber17 committed:
Update FaceAPIinJavaForAndroidTutorial.md
